modules/styling/lib/cs.ts

```diff
@@ -733,16 +733,16 @@ export function handleCsProp<
 
 type StylesReturn<V extends Record<string, string> | Record<string, Record<string, string>> = {}> =
   | SerializedStyles
-  | CSSObject
-  | ((vars: OptionalVars<V>) => SerializedStyles | CSSObject);
+  | CSSObjectWithVars
+  | ((vars: OptionalVars<V>) => SerializedStyles | CSSObjectWithVars);
 
 export type StencilModifierConfig<
   V extends Record<string, string> | Record<string, Record<string, string>> = {}
 > = Record<string, Record<string, StylesReturn<V>>>;
 
 export type StencilCompoundConfig<M> = {
-  modifiers: {[K in keyof M]?: MaybeBoolean<keyof M[K]>};
-  styles: SerializedStyles | CSSObject;
+  modifiers: {[K in keyof M]?: keyof M[K]};
+  styles: SerializedStyles | CSSObjectWithVars;
```

Comment:
Did you mean to remove the `MaybeBoolean` here?

Reply:
Seems like my file hasn't updated 🤔 I pushed the fix

Reply:
It should be only `CSSObject` replacements

Reply:
This looks updated and now has the `MaybeBeelean`